### PR TITLE
Type about

### PR DESCRIPTION
**Pull request by chloeleichen**
### Checklist

-   [x] There are unit tests to verify my changes are correct | Unit tests aren't applicable (delete one)
-   [x] I've updated CHANGES.md with what I changed.
-   [x] I've linked this PR to an issue in ZenHub (or there is no issue) and dragged it to the _Pull Request_ column
-   [x] There are sufficient comments for my code to be understandable - and I realise reviewers will pull me up on it if not!

### CODE REVIEW

**Comment by chloeleichen** [comment]
@tkeuneman please review content here: https://github.com/magda-io/magda/blob/d7e1f547868550bb0173825f029389a419ac2a2f/magda-web-client/src/markdown/about.md

**Comment by tkeuneman** [comment]
Reviewing with @maxious the section about being able to purchase datasets. 

Alex, can you confirm that we should have a link to sign in as well? Seems odd to not mention it in the 'registering for the first time' section.

Can you remove the link in this sentence: Our Open Data Toolkit also includes useful information and links to other open data sources, on topics like how to form a policy about sharing data through to creating a basic data set.

File: magda-web-client/src/markdown/about.md [hunk 1]
```diff
@@ -26,15 +26,8 @@ It is important you fill out your profile completely, including your agency’s
...
-
-## Want to keep in touch?
-
-Subscribe to the new datasets feed or customise a feed below:
-[Groups](https://data.gov.au/feeds/group/),
-[Tags](https://data.gov.au/feeds/tag/) or
-[Search](https://data.gov.au/feeds/custom.atom?q=).
+Many agencies are excited but nervous the first time their share data. You should always consider privacy and cleanliness of data before sharing. Information regarding privacy can be found in our Privacy Policy and in our [Open Data Toolkit](https://toolkit.data.gov.au/index.php?title=Main_Page). Our [Open Data Toolkit](https://toolkit.data.gov.au/index.php?title=Main_Page) also includes useful information and links to other open data sources, on topics like how to form a policy about sharing data through to creating a basic data set.
```
Comment (maxious) [inline magda-web-client/src/markdown/about.md]: >the first time their share data. 

SHouldn't that be "the first time they are sharing their data. "

**Comment by maxious** [comment]
Maybe for the "registering for the first time", first sentence second paragraph should be "Enter this information at to (https://data.gov.au/user/register)[create your account.]"

File: magda-web-client/src/markdown/about.md (hunk 1 above)
Comment (tkeuneman) [reply magda-web-client/src/markdown/about.md]: Sure. Once that's changed, we're good to go.

**Comment by chloeleichen** [comment]
Updated

**Pull request merged**